packages/web/components/TechnologyForm/MapAndAttachments/index.js

```diff
@@ -28,8 +28,10 @@ import {
 import { Row, Column } from '../../Common/Layout';
 import { CircularButton } from '../../Button';
 
-const parseMetaObjectIntoKeyValue = (findTerm, terms) => {
-	const filteredTerms = terms.filter(({ term }) => term === findTerm);
+const parseMetaObjectIntoKeyValue = (findTerm, terms, taxonomy) => {
+	const filteredTerms = terms.filter(
+		(term) => term.term === findTerm && term.taxonomy.taxonomy === taxonomy,
+	);
```

Comment:
here we could have different taxonomies coming from rawTerms, we need to filter based on `taxonomy`